dsls/sql/src/main/java/org/apache/beam/dsls/sql/transform/BeamAggregationTransforms.java

```diff
@@ -59,13 +59,16 @@
   public static class MergeAggregationRecord extends DoFn<KV<BeamSqlRow, BeamSqlRow>, BeamSqlRow> {
     private BeamSqlRecordType outRecordType;
     private List<String> aggFieldNames;
+    private int windowFieldIdx;
```

Comment:
Two comments:
1. Name this windowStartFieldIdx or startFieldIdx, since there will presumably be a similar index for the end of the window eventually?
2. Use Integer and have null be the special value for no such field? Seems a little more conventional than an arbitrary negative index.